auth/user.js

```diff
@@ -13,3 +14,27 @@ if (!user) {
 }
 
 signOutLink.addEventListener('click', signOutUser);
+
+let error = null;
+let profile = null;
+
+window.addEventListener('load', async () => {
+    const response = await getProfile(user.id);
+    error = response.error;
+    profile = response.data;
+
+    if (error) {
+        console.log(error);
+        return;
+    }
+
+    if (profile) {
+        displayProfile();
+    }
+});
+
+function displayProfile() {
+    if (userAvatar && profile && profile.avatar_url) {
+        userAvatar.src = profile.avatar_url;
+    }
```

Comment:
real nit-picky but you could leave out the check for ```profile``` because if ```profile.avatar_url``` is ```true``` than ```profile``` is by default going to be true